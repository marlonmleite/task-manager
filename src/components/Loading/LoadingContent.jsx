import React from 'react'
import { Loading, LoadingContent as LoadingContentStyled } from './styled'

const LoadingContent = () => (
  <LoadingContentStyled>
    <Loading />
  </LoadingContentStyled>
)

export default LoadingContent
