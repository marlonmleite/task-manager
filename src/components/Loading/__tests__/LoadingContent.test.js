import React from 'react'
import { shallow } from 'enzyme'
import LoadingContent from '../LoadingContent'

describe('LoadingContent', () => {
  it('should render', () => {
    const loading = shallow(<LoadingContent />)

    expect(loading).toMatchSnapshot()
  })
})
