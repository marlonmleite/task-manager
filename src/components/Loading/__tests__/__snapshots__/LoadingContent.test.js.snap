// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingContent should render 1`] = `
<styled.div>
  <styled.div
    color="primary"
    size="75px"
  />
</styled.div>
`;
